src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -255,6 +263,26 @@
 								<template #icon>
 									<ConversationIcon :item="iconData(item)" />
 								</template>
+								<template #subname>
+									{{ t('spreed', 'New group conversation') }}
+								</template>
+							</NcListItem>
+						</template>
+
+						<!-- New conversations: Federated users -->
+						<template v-if="searchResultsFederated.length !== 0">
+							<NcAppNavigationCaption :name="t('spreed', 'Federated users')" />
+							<NcListItem v-for="item of searchResultsFederated"
+								:key="`circle_${item.id}`"
+								:data-nav-id="`circle_${item.id}`"
+								:name="item.label"
+								@click="createAndJoinConversation(item)">
+								<template #icon>
+									<AvatarWrapper v-bind="iconData(item)" />
```

Comment:
`conversationIcon` ? like users

Reply:
It doesn't contain logic for fetching proxy users (and I don't want to add it there)